### DISCUSSION TITLE
是否可以来个Net的最佳实践？

### DISCUSSION THREAD

**Discussion by Aaronphy** [liangjingkanji/Net#181] [Q&A] [answered]
因为纯碎看文档 没有sample能把他们一起串起来的深刻

**Comment by liangjingkanji** [comment, marked as answer]
Net不是有sample吗? 如果觉得没有涉及到UI你可以去看BRV的sample

**Reply by Aaronphy** [reply]
没注意项目里有sample这个目录 谢谢 已经找到

**Reply by liangjingkanji** [reply]
完全没下载过项目是吧

**Reply by Aaronphy** [reply]
在看掘金的时候看了这篇文章，没有仔细看项目结构，实在抱歉，现在已经在用了，发现用起来太方便了！！
再多请教一个问题，目前我开发的App 是依赖webview登录的，Net可以和WebView 共享cookie？还是得通过cookiemanager 去取cookie然后放在header里？

**Reply by liangjingkanji** [reply]
和OkHttp一样的处理方式, 只不过Net有提供一个默认使用数据库实现好的序列化Cookie管理器(实现OkHttp.cookieJar接口)

Net基于OkHttp封装只要OkHttp可以实现就可以, 而且本项目都说了是OkHttp的最佳实践案例
